src/RpcServer/RpcServer.Wallet.cs

```diff
@@ -229,6 +229,113 @@ protected virtual JToken SendFrom(JArray _params)
+            }
+            if (!wallet.Sign(context))
+            {
+                throw new RpcException(-100, $"Non-existent private key in wallet.");
+            }
+            var json = new JObject();
+            var tx = (Transaction)context.Verifiable;
+            json["sign_context"] = context.ToJson();
+            if (context.Completed)
+            {
+                tx.Witnesses = context.GetWitnesses();
+                json["tx_raw"] = Convert.ToBase64String(tx.ToArray());
+            }
+            tx.Witnesses ??= Array.Empty<Witness>();
+            json["tx"] = tx.ToJson(system.Settings);
+            return json;
+        }
+
+        private JObject SendWithoutSign(UInt160 assetId, UInt160 from, UInt160 to, BigInteger amount, Signer[] signers, long? networkFee = null, long? sysFee = null, uint? untilBlock = null)
+        {
+            using var snapshot = system.GetSnapshot();
+            var sb = new ScriptBuilder();
+            sb.EmitDynamicCall(assetId, "transfer", from, to, amount, null);
+            sb.Emit(OpCode.ASSERT);
+            var script = sb.ToArray();
+            Random rand = new();
+            Transaction tx = new Transaction()
+            {
+                Attributes = Array.Empty<TransactionAttribute>(),
+                Nonce = (uint)rand.Next(),
+                Signers = signers,
+                Script = script,
+                Witnesses = Array.Empty<Witness>(),
+                ValidUntilBlock = untilBlock ?? NativeContract.Ledger.CurrentIndex(snapshot) + system.Settings.MaxValidUntilBlockIncrement,
+            };
+
+            if (sysFee.HasValue)
+            {
+                tx.SystemFee = sysFee.Value;
+            }
+            else
+            {
+                // will try to execute 'transfer' script to check if it works 
+                using ApplicationEngine engine = ApplicationEngine.Run(script, snapshot.CreateSnapshot(), tx, settings: system.Settings, gas: ApplicationEngine.TestModeGas);
+                if (engine.State == VMState.FAULT)
+                {
+                    throw new InvalidOperationException($"Failed execution for '{script.ToHexString()}'", engine.FaultException);
+                }
+                tx.SystemFee = engine.GasConsumed;
+            }
+            tx.NetworkFee = networkFee ?? QuickCalculateNetworkFee(system.StoreView, tx);
+
+            ContractParametersContext transContext = new(snapshot, tx, settings.Network);
+            var json = new JObject();
+            json["sign_context"] = transContext.ToJson();
+            json["tx"] = tx.ToJson(system.Settings);
+            return json;
+        }
+
+        /// <summary>
+        /// treat signer as single Signature
+        /// </summary>
+        /// <param name="snapshot"></param>
+        /// <param name="tx"></param>
+        /// <returns></returns>
+        private long QuickCalculateNetworkFee(DataCache snapshot, Transaction tx)
```

Comment:
But why? `calculatenetworkfee` implementation does it this way:
```
            long networkfee = (wallet ?? new DummyWallet(system.Settings)).CalculateNetworkFee(system.StoreView, tx.AsSerializable<Transaction>());
```
And if you're only support using one simple account you might as well create some fake `VerificationScript` for it.

Reply:
DummyWallet won't work here.  The result tx could be sign and relay, it's not logical to set a fake script.

Reply:
You can set it, calculate the fee and then remove again. I'd worry more about code duplication (12 case-specific lines) rather than two lines of additional code.

Reply:
How to make [this line](https://github.com/neo-project/neo/blob/5f4c14e3801c691ceccb5705c634e9e55dd3bba1/src/Neo/Wallets/Wallet.cs#L602) work? It couldn't be just 2 lines.

Reply:
Use fake verification script for https://github.com/neo-project/neo/blob/5f4c14e3801c691ceccb5705c634e9e55dd3bba1/src/Neo/Wallets/Wallet.cs#L609-L612 ?